Lib/test/test_codeop.py

```diff
@@ -160,7 +160,6 @@ def test_incomplete(self):
         ai("","eval")
         ai("\n","eval")
         ai("(","eval")
-        ai("(\n\n\n","eval")
```

Comment:
The EOF was reached in a different line before but now it raises the same error as the case without newlines. I think we can allow that to happen.